src/main/java/grid/Hub.java

```diff
@@ -0,0 +1,10 @@
+package grid;
+
+import org.openqa.grid.selenium.GridLauncherV3;
+
+public class Hub {
+
+    public static void main(String[] args) {
```

Comment:
If you decided to launch Hub and Nodes from code, from my perspective it is much preferable to do it usign Json config file.